packages/server/src/routes/members/index.ts

```diff
@@ -64,26 +64,26 @@ function getPrivateUserOnlyVideoRef(
     .file(`private-video/${memberUid}/invite.mp4`);
 }
 
-function getPrivateInviteVideoRef(
+function getPublicVideoBucketRef(config: Config, storage: BucketStorage) {
+  return (storage as Storage.Storage).bucket(config.publicVideoBucket);
+}
+
+function getPublicInviteVideoRef(
   config: Config,
   storage: BucketStorage,
-  videoToken: string
+  uid: string
```

Comment:
@rahulgi  this is not a mistaken revert to `uid`. It's that `getPrivateInviteVideoRef` got removed because it was the only use of that dead function, but `getPublicInviteVideoRef` does still use `uid` since it indexes invite videos based on `uid` (the TODO comment in `shared/` regarding justtttt having video urls for `create_member` operations would obviate this function).    Just the way the diff gets displayed makes it look like that.